grabbit/src/main/resources/com/twcable/grabbit/client/servlets/RootResource.txt

```diff
@@ -0,0 +1,3 @@
+Grabbit Does not support direct requests on this url.
+Visit one of the following links for more info!
```

Comment:
Why not just store the static HTML file describing the child resources like you were before?